lib/classes/ssl/class.lescript.php

```diff
@@ -494,6 +493,35 @@ protected function log($message)
 	{
 		$this->logger->logAction(CRON_ACTION, LOG_INFO, "letsencrypt " . $message);
 	}
+
+    /**
+     * Ask PHP Curl do fetch data from given URL.
+     *
+     * @param string $url - The URL to grab data from
+     *
+     * @return mixed
+     */
+    protected function getUrlContent($url){
+        $r = [
+            'error' => TRUE,
+            'error_message' => '',
+            'data' => '',
+        ];
+        $ch = curl_init();
+        curl_setopt($ch, CURLOPT_URL, $url);
+        curl_setopt($ch, CURLOPT_USERAGENT, 'Mozilla/4.0 (compatible; MSIE 6.0; Windows NT 5.1; .NET CLR 1.1.4322)');
```

Comment:
You even patched it away, it was in there already: https://github.com/Froxlor/Froxlor/blob/master/lib/classes/ssl/class.lescript.php#L230